docs/pages/machine-id/guides/github-actions.mdx

```diff
@@ -0,0 +1,173 @@
+---
+title: Using Machine ID With Github Actions
+description: A tutorial for using Machinie ID with Github Actions
+---
+
+Github actions are a popular CI/CD platform that works as a larger part of the Github
+ecosystem. Teleport, with the help of Machine ID, allows for Github actions to
+securely interact with Teleport Protected resources without the need for long lived
+credentials. By ensuring that rotated Machine ID credentials are additionally limited by RBAC,
+Github actions with Teleport can be applied as both a viable and secure solution whenever a
+CI/CD platform is needed.
+
+## Prerequisites
```

Comment:
We should mention that their Teleport proxy needs to be accessible from GitHub Actions